app/db/AddIndexIfNotExists.sql

```diff
@@ -14,7 +13,7 @@ CREATE PROCEDURE AddIndexIfNotExists(
 			and table_schema=dbName
 			)
 		THEN
-			SET @ddl=CONCAT('CREATE ',indexType,' INDEX `',indexName,'` ON `',dbName,'`.`',tableName,'` (',indexFields,')');
```

Comment:
I believe `indexType` is useful whenever you need to create a `UNIQUE` index ->
`CREATE UNIQUE INDEX`